.github/jwt.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
Source: https://docs.github.com/en/apps/creating-github-apps/authenticating-with-a-github-app/generating-a-json-web-token-jwt-for-a-github-app

Reply:
Maybe the link to the source of this file, that you put in this GitHub comment, should be put directly as a comment in the file itself, so that we don’t forget that it comes verbatim from GitHub doc.